### ISSUE TITLE
Warning and Critical filters not working 

### ISSUE THREAD

**Issue by 0x412e4e** [matteocorti/check_updates#39]
**Describe the bug**

Running `./check_updates` with a warning threshold of 40 and a critical threshold of 50 reports having 7 available updates as `CHECK_UPDATES CRITICAL`

**To Reproduce**

`./check_updates --clean -t 120 -w 40 -c 50`

**Expected behavior**

```[root@centos]# ./check_updates --clean -t 120 -w 40 -c 50
CHECK_UPDATES OK - 1 security update and 6 non-security updates available | total_updates=7;40;50               security_updates=1;40;50
openssl11-libs.x86_64 (security)
kernel.x86_64
kernel-debug-devel.x86_64
kernel-headers.x86_64
kernel-tools.x86_64
kernel-tools-libs.x86_64
python-perf.x86_64
```

**Current behavior**

```[root@centos]# /usr/lib64/nagios/plugins/check_updates --clean -t 120 -w 40 -c 50
CHECK_UPDATES CRITICAL - 1 security update and 6 non-security updates available | total_updates=7;40;50               security_updates=1;40;50
openssl11-libs.x86_64 (security)
kernel.x86_64
kernel-debug-devel.x86_64
kernel-headers.x86_64
kernel-tools.x86_64
kernel-tools-libs.x86_64
python-perf.x86_64
```

**System (please complete the following information):**

 - OS: CentOS
 - OS Version: 7.9.2009 (Core)
 - check_updates Version: 1.7.13

**Additional context/output**

`check_updates -d -v` output:

```[DBG] check_updates version: 1.7.13
[DBG] system info:           Linux centos 3.10.0-1160.21.1.el7.x86_64 #1 SMP Tue Mar 16 18:28:22 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
[DBG]                        NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

Checking a linux system
Running on CentOS Linux release 7.9.2009 (Core)
[DBG] Updater: yum
Using Yum or DNF
running a Linux kernel: 3.10.0-1160.21.1
kernel: running = 3.10.0-1160.21.1, installed = 3.10.0-1160.21.1
[DBG] Running "/usr/bin/yum check-update --assumeno --errorlevel=0 -q  2>&1"
[DBG]
[DBG] kernel.x86_64                          3.10.0-1160.24.1.el7              updates
[DBG] kernel-debug-devel.x86_64              3.10.0-1160.24.1.el7              updates
[DBG] kernel-headers.x86_64                  3.10.0-1160.24.1.el7              updates
[DBG] kernel-tools.x86_64                    3.10.0-1160.24.1.el7              updates
[DBG] kernel-tools-libs.x86_64               3.10.0-1160.24.1.el7              updates
[DBG] openssl11-libs.x86_64                  1:1.1.1g-3.el7                    epel
[DBG] python-perf.x86_64                     3.10.0-1160.24.1.el7              updates
Security updates available
[DBG] Running "/usr/bin/yum check-update --assumeno --errorlevel=0 -q  --security 2>&1 2>&1"
[DBG]
[DBG] openssl11-libs.x86_64                    1:1.1.1g-3.el7                     epel
Security updates available
CHECK_UPDATES CRITICAL - 1 security update and 6 non-security updates available | total_updates=7;0;0 security_updates=1;0;0
openssl11-libs.x86_64 (security)
kernel.x86_64
kernel-debug-devel.x86_64
kernel-headers.x86_64
kernel-tools.x86_64
kernel-tools-libs.x86_64
python-perf.x86_64```

**Comment by matteocorti** [comment]
The plugin always returns a critical status if at least one security option is available.
I see that this is not properly documented.

**Comment by 0x412e4e** [comment]
Would be a nice feature to override that.

My issue is that I'm monitoring a dozen CentOS server's update statuses, and when even a single security update is made available I get a ton of useless alerts, even if the actual update thresholds are not met.

**Comment by matteocorti** [comment]
I agree, I'll add an option

**Comment by matteocorti** [comment]
Any suggestion for a good option name?

**Comment by 0x412e4e** [comment]
Hmm... perhaps something like "override default security update threshold"?

**Comment by matteocorti** [comment]
A little long ... :-)

**Comment by matteocorti** [comment]
Something like --count-only? --number-only?

**Comment by matteocorti** [comment]
Fixed in bd80c3b
